deployments/cloudfoundry/tile/README.md

```diff
@@ -3,24 +3,32 @@
 This directory is used to generate a Tanzu tile of the [Splunk OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector).
 The Tanzu tile uses the BOSH release to deploy the collector as a [loggregator firehose nozzle](https://docs.vmware.com/en/VMware-Tanzu-Operations-Manager/3.0/tile-dev-guide/nozzle.html).
 
-## Dependencies
+## CI Build and Artifacts
 
-The `release` script requires:
+If you just want to publish a new version of the tile, you can use the artifacts of the
+[Tanzu Tile](../../../.github/workflows/tanzu-tile.yml) GitHub Action workflow.
 
-- `jq`
-- `wget`
-- [Bosh CLI](https://bosh.io/docs/cli-v2-install/)
-- [Tile Generator](https://docs.vmware.com/en/VMware-Tanzu-Operations-Manager/3.0/tile-dev-guide/tile-generator.html) -
-Note that MacOS support was dropped for this tool, so an older version must be downloaded for darwin development.
-Version `14.0.6-dev.1` has been confirmed to be work.
+The artifacts of the workflow include:
 
-## Releasing
+- The generated tile file: `splunk-otel-collector-<VERSION>.pivotal` at
+  `deployments/cloudfoundry/tile/product/`
+- The OSDF file, `OSDF V<VERSION>.txt` at the artifact root.
```

Comment:
This is not exactly true, there is a small bug in which the artifact file is named `OSDF Vv<VERSION>.txt`, but since there is already a Jira ticket to fix it and perform other CI improvements related to Tanzu I opted to use the "correct" name here.